lizmap/www/assets/js/attributeTable.js

```diff
@@ -1462,6 +1463,32 @@ var lizAttributeTable = function() {
                         }
                     }
 
+                    var newcolumnsfinal = lizcols.concat(newcolumns)
+                    colToReturn['columns'] = newcolumnsfinal;
+                } else if (
+                    'columns' in config.layers[aName]
+                    && config.layers[aName]['columns']
+                    && !$.isEmptyObject(config.layers[aName]['columns'])
```

Comment:
```suggestion
                    && Object.keys(config.layers[aName]['columns']).length > 0)
```

Reply:
We should stop using jQuery.

Reply:
@nboisteault I have to learn more about new JavaScript ;-)